apps/www/registry/tui/example/breadcrumbIcon.tsx

```diff
@@ -18,18 +18,19 @@ export const BreadcrumbIcon = () => {
     ]
     return (
         <>
-            <h2 className="text-m text-primary pt-3">BreadCrumb With Icon</h2>
-            <div className="text-m font-bold text-primary">Contained</div>
-            <Breadcrumbs dataItem={firstItem} contained={true} iconStyle="h-5 w-5 text-gray-500" separatorIconStyle="h-full w-6 flex-shrink-0 text-gray-200" textColor="gray" />
+            <h2 className="text-m text-primary pt-11">BreadCrumb With Icon</h2>
 
-            <div className="text-m pt-11 font-bold text-primary">Full Width Bar</div>
-            <Breadcrumbs dataItem={firstItem} fullWidthContained={true} iconStyle="h-5 w-5 text-gray-500" separatorIconStyle="h-full w-6 flex-shrink-0 text-gray-200" textColor="gray" size="default" />
+            <div className="text-m font-bold text-primary pt-2 pb-6">Contained</div>
+            <Breadcrumbs dataItem={firstItem} contained={true} iconStyle="h-5 w-5 text-gray-400 hover:text-gray-500" separatorIconStyle="h-full w-6 flex-shrink-0 text-gray-200" textColor="gray" />
 
-            <div className="text-m pt-11 font-bold text-primary">Simple With Chevron</div>
-            <Breadcrumbs dataItem={secondItem} withChevronSeparator={true} iconStyle="h-5 w-5 text-gray-500" separatorIconStyle="h-4 w-4 flex-shrink-0 text-gray-400" textColor="gray" />
+            <div className="text-m pt-11 font-bold text-primary pb-6">Full Width Bar</div>
```

Comment:
what's text-m ? I see its been used in BreadCrumb, Button Group, Pagination, sidebar navigation, steps, text-area, notification